geschaeftsordnung-vollversammlung.md

```diff
@@ -0,0 +1,82 @@
+
+# Geschäftsordnung der Vollversammlung der Fachschaft Digital Engineering
+
+Diese Geschäftsordnung wurde am xx.xx.xxxx von der Vollversammlung beschlossen.
+
+## § 1 Geltungsbereich
+
+Die Geschäftsordnung gilt für die Vollversammlung der Fachschaft Digital Engineering an der Universität Potsdam.
+
+## § 2 Tagesordnung
+
+(1) Die Vollversammlung folgt einer Tagesordnung.
+
+(2) Die Tagesordnung muss Teil der fristgerechten Einladung einer Vollversammlung sein.
+
+(3) Änderungsanträge zur Tagesordnung einer ordentlichen Vollversammlung können bis zu 14 Tage vor der Vollversammlung bei den Einladenden gestellt werden.
+
+(4) 7 Tage vor einer ordentlichen Vollversammlung muss die finale Tagesordnung auf dem gleichen Weg wie die Einladung zur Vollversammlung bekanntgegeben werden.
+
+## § 3 Versammlungsleitung
+
+(1) Die Vollversammlung wird von den Einladenden geleitet.
+
+(2) Die Versammlungsleitung kann für einzelne Tagesordnungspunkte an andere Mitglieder der Fachschaft übergeben werden.
+
+(3) Die Versammlungsleitung kann bei Bedarf eine Redeliste führen und / oder die Redezeit begrenzen.
+
+(4) Die Versammlungsleitung entscheidet über die explizite Zählung und Protokollierung von Stimmen.
+
+## § 4 Protokolle
+
+(1) Die Vollversammlung dokumentiert ihre Beschlüsse in einem Protokoll.
+
+(2) Die Versammlungsleitung beauftragt ein Mitglied der Fachschaft mit der Protokollführung.
+
+(3) Das Protokoll wird der Fachschaft spätestens 7 Tage nach der Vollversammlung zugänglich gemacht.
+
+## $ 5 Anträge
+
+Anträge können vor der Behandlung durch die antragstellende Person zurückgezogen werden.
+
+## § 6 Änderungsanträge
+
+(1) Änderungsanträge sind Anträge zur Änderung der Tagesordnung oder zur Änderung von Anträgen.
+
+(2) Änderungsanträge werden vor den jeweiligen Anträgen behandelt.
+
+(3) Änderungsanträge können nur von der den ursprünglichen Antrag stellenden Person direkt angenommen werden. Geschieht dies nicht, stimmt die Vollversammlung über die Annahme ab.
+
+(4) Änderungsanträge können vor der Vollversammlung schriftlich bekanntgegeben und begründet werden.
+
+## § 7 Anträge zur Geschäftsordnung
+
+(1) Anträge zur Geschäftsordnung werden durch das Heben beider Arme oder anderweitig eindeutig angezeigt.
+
+(2) Anträge zur Geschäftsordnung werden vor Änderungsanträgen behandelt.
+
+(3) Angenommene Geschäftsordnungsanträge stehen über den Bestimmungen dieser Geschäftsordnung.
+
+(4) Geschäftsordnungsanträge bedürfen einer Zweidrittelmehrheit der anwesenden Mitglieder der Vollversammlung.
```

Comment:
"..., soweit nicht explizit andere Quoren festgelegt sind."

Reply:
Obwohl das in (5) bereits steht? Finde das gefährlich. Jetzt könnte man "aus Versehen" woanders ein anderes Quorum definieren, ohne zu erwähnen, dass es § 7 (5) überschreibt

Reply:
ich verstehe nicht, was du meinst. was hat (5) damit zu tun? ich würde hier definieren, was grundsätzlich für eine mehrheit erforderlich ist und dann in (6) in der auflistung in klammern hinter einzelne punkte eventuell abweichende mehrheitsregelungen.

Reply:
Ich glaube die Nummerierung der Absätze hat sich vor wenigen Tagen geändert.

(4) regelt den Default und in (6) stehen bereits andere Quoren und der Vermerk, dass sie (4) überschreiben: "Die hier genannten Quoren gelten abweichend zu § 7 (4)". Das meinte ich mit dem Kommentar von vor 9 Tagen